pnt-lut.c

```diff
@@ -62,21 +68,36 @@ int main(int ac, char *av[]) {
     mkfifo(pnt_fifo, 0666);
     const int fd = open(pnt_fifo, O_RDONLY);
     const int fd_out = open(output_fifo, O_WRONLY);
-    uint8_t pnt[2];
+    int pnt[2];
```

Comment:
why signed?

Reply:
they're all signed, ascii-conv outputs signed. just compatibility reasons here, but as an overall design decision it's nice to have the option to use the converter for signed stuff too. We never go over the max int value so it doesn't really matter otherwise